src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -245,13 +258,6 @@ class CassandraJournal(cfg: Config) extends AsyncWriteJournal with CassandraReco
         bs.setString("event_manifest", m.eventManifest)
         bs.setBytes("event", m.serialized)
 
-        if (session.protocolVersion.compareTo(ProtocolVersion.V4) < 0) {
-          bs.setToNull("message")
-          (1 to maxTagsPerEvent).foreach(tagId => bs.setToNull("tag" + tagId))
-        } else {
-          bs.unset("message")
-        }
-
         if (m.tags.nonEmpty) {
```

Comment:
and we want to make sure that this is only done when `CassandraJournalConfig.enableEventsByTagQuery` is `true`, but I think it's better to check `enableEventsByTagQuery` in the `serialize` method so that tags are not populated at all if `enableEventsByTagQuery` is `false`